npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher/path_dependency_builder.rb

```diff
@@ -120,8 +120,8 @@ def parsed_yarn_lock
               File.write("yarn.lock", yarn_lock.content)
 
               SharedHelpers.run_helper_subprocess(
-                command: "node #{yarn_helper_path}",
-                function: "parseLockfile",
+                command: NativeHelpers.helper_path,
+                function: "yarn_parseLockfile",
```

Comment:
Matching the function to the map

```suggestion
                function: "yarn:parseLockfile",
```